src/ServiceProvider.php

```diff
@@ -42,4 +43,27 @@ protected function bootAddonViews()
 
         return $this;
     }
+
+    protected function bootAddonConfig()
+    {
+        $this->publishes([
+            __DIR__.'/../config/responsive-images.php' => config_path('statamic/responsive-images.php'),
+        ], 'responsive-images-config');
+
+        return $this;
+    }
+
+    public function register() {
+        parent::register();
+
+        $this->registerAddonConfig();
+    }
+
+    protected function registerAddonConfig() {
```

Comment:
This doesn't look correct, it can also be added to the `bootAddonConfig` method instead, take a look at https://github.com/statamic/seo-pro/blob/master/src/ServiceProvider.php#L50-L59 for an example